scripts/app/test_internal_chain.py

```diff
@@ -87,3 +89,9 @@ async def process_block(blocks):
 
             # Increment the offset for the next query
             offset += current_limit
+
+        # The URL you want to send the data to
+        url = f'https://planetariumhq.slack.com/services/hooks/slackbot?token={config.slack_token}&channel=%239c-internal'
+        data = f"[9C-INFRA] Finished testing `{network}` network from `#{original_offset}` to `#{tip_index}`."
+        headers = {'Content-Type': 'text/plain'}
+        response = requests.post(url, data=data, headers=headers)
```

Comment:
should run only if config.slack_token exists
channel is hardcoded to 9c-internal

Reply:
I've updated the code to take the channel name as an environment variable 🙏